src/components/button/button.ts

```diff
@@ -32,49 +43,55 @@ export class MdButton {
     setTimeout(() => { this.isMouseDown = false; }, 100);
```

Comment:
I think you can get rid of this `setTimeout` logic by, on the focus event, checking `event instanceof KeyboardEvent`.